src/test/Testcafe/TestData.js

```diff
@@ -5,25 +5,37 @@ import config from './config';
 
 let app = new App();
 let loginPage = new LoginPage();
-let environment = (process.env.ENV).toLowerCase();
+let environment = process.env.ENV.toLowerCase();
```

Comment:
I'm thinking that changing the casing of these variables in the test might hide a typo.
Shouldn't the environment always be in the same casing everywhere?

Reply:
`ENV` is set by the user in cmd before running the tests, so the user might use upper or lower case